shells/default/default.nix

```diff
@@ -45,11 +46,17 @@
       commitizen
       alejandra
       shfmt
-      mitmproxy
       yq
       cargo-nextest
       procps
       pstree
+    ]
+    ++ lib.optionals stdenv.isLinux [
+      # The python3Packages.mitmproxy-macos package is broken on mac:
+      #   nix-repl> legacyPackages.aarch64-darwin.python3Packages.mitmproxy-macos.meta.broken
+      #   true
+      # ... so only install it on Linux. It's only an optional dev dependency.
+      mitmproxy
```

Comment:
I've used this frequently to debug the catalog, metrics, and Sentry clients. I could go back to using `nix run …` from `nixpkgs` but think it would be worth updating the instructions [here](https://github.com/flox/flox/blob/eabea829aa88bd597c87f229e74a6e310db8d1a6/cli/catalog-api-v1/README.md#debugging) so that everyone can use it consistently.

Reply:
Updated doc in https://github.com/flox/flox/pull/2030

Took awhile to find the most recent nixpkgs stable tag where `mitmproxy` worked on darwin.  :-\

Resolving conversation